dropbox/package.json

```diff
@@ -1,20 +1,23 @@
 {
-    "name"          : "node-red-node-dropbox",
-    "version"       : "1.0.3",
-    "description"   : "Node-RED nodes to watch and download files from Dropbox",
-    "dependencies"  : {
-        "dropbox":"^2.5.10",
-        "minimatch":"^3.0.4",
-        "is-utf8":"^0.2.1"
+    "name": "node-red-node-dropbox",
+    "version": "1.0.4",
```

Comment:
Probably worth a 1.1.0 given the large version jumps.... if not a 2.0.0

Reply:
1.1.0 then so it's more likely to get auto upgraded on a rebuild